rep-2008.rst

```diff
@@ -0,0 +1,777 @@
+    * - Kernel Levels
+      - 
+      - 
+      - 
+      - 
+    * - `1.i`_
+      - ✓
+      - ✓
+      - ✓
+      - 
+    * - 1.ii_
+      - ●
+      - ●
+      - ○
+      -
+    * - 1.iii_
+      - ●
+      - ○
+      - ○
+      -
+    * - Build System
+      - 
+      - 
+      - 
+      -     
+    * - 2.i_
+      - ✓
+      - ✓
+      - ✓
+      - 
+    * - 2.ii_
+      - ●
+      - ●
+      - ○
+      -
+    * - Build Tools
+      - 
+      - 
+      - 
+      -     
+    * - 3.i_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.ii_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.iii_
+      - ✓
+      - ✓
+      - ●
+      - 
+    * - 3.iv_
+      - ✓
+      - ✓
+      - ○
+      -
+    * - 3.iv.a_
+      - ✓
+      - ●
+      - ○
+      -
+    * - 3.iv.b_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.v_
+      - ✓
+      - ●
+      - ○
+      -
+    * - 3.v.a_
+      - ✓
+      - ●
+      - ○
+      -
+    * - 3.v.b_
+      - ✓
+      - ●
+      - ○
+      -
+    * - 3.v.c_
+      - ✓
+      - ●
+      - ○
+      -
+    * - 3.v.d_
+      - ✓
+      - ●
+      - ○
+      -
+    * - 3.vi_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.vi.a_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.vi.b_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.vi.c_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.vi.d_
+      - ●
+      - ●
+      - ○
+      -
+    * - 3.vi.e_
+      - ●
+      - ●
+      - ○
+      -
+    * - Benchmarking
+      - 
+      - 
+      - 
+      -     
+    * - 4.i_
+      - ●
+      - ●
+      - ○
+      -
+    * - 4.ii_
+      - ●
+      - ●
+      - ○
+      -
+    * - Documentation
+      - 
+      - 
+      - 
+      -     
+    * - 5.i_
+      - ✓
+      - ✓
+      - ●
+      - 
+    * - Testing and CI
+      - 
+      - 
+      - 
+      -     
+    * - 6.i_
+      - ✓
+      - ●
+      - ○
+      - 
+
+
+
+Reference Implementation and recommendations
+============================================
+
+Reference implementations complying with this REP and extending the ROS 2 build system and tools for hardware acceleration are available at the `Hardware Accelerationg WG GitHub organization <https://github.com/ros-acceleration>`_. This includes also the abstraction layer ``ament_acceleration`` and firmware from vendor specializalizations like `ament_vitis <https://github.com/ros-acceleration/ament_vitis/>`_.
+
+For additional implementations and recommendations, check out the `Hardware Accelerationg WG GitHub organization <https://github.com/ros-acceleration>`_.
+
+
+
+.. _labels for maintainers:
+
+Labels for maintainers
+^^^^^^^^^^^^^^^^^^^^^^
+Maintainers are encouraged to mark their packages including hardware acceleration with a label that indicates so, for each one of the hardware acceleration technology solutions they support. This REP proposes that such indication takes the following shape:
+
+.. image:: https://img.shields.io/badge/hardware_acceleration-KV260-ec1c24.svg
```

Comment:
I just browsed some packages and don't see these labels commonly used within the ROS ecosystem. I'm not sure if there's a precedent for this sort of thing in REPs.

Reply:
You're right, there's no precedent (that I know at least) of using *shields* in REPs.

That said, they're widely used in some of the most popular packages (e.g. [navigation2](https://github.com/ros-planning/navigation2), [moveit2](https://github.com/ros-planning/moveit2), [ros2_control](https://github.com/ros-controls/ros2_control), to mention a few). Given that most maintainers are picking *shields*, I believe it's somewhat fair to encourage them to do so. That's what's attempt in here.

Reply:
I believe we've addressed this by now since the content has dissappeared from the REP.